components/base/Collapsible.tsx

```diff
@@ -111,7 +115,13 @@ function Collapsible({ triggerText, children }: Props) {
         transitionTime={transitionTime}
         open={active}
       >
-        <div tabIndex={active ? 0 : -1}>{children}</div>
+        <InnerContentWrapper
+          className={classnames({
+            visuallyhidden: !active,
+          })}
+        >
+          {children}
+        </InnerContentWrapper>
```

Comment:
Collapsible innerContent is now visually hidden when not opened. This disables the links/content from being tabbable when the Collapsible is closed.